pymeasure/instruments/srs/sr860.py

```diff
@@ -0,0 +1,519 @@
+        Volts, taking values between -10.5 V and +10.5 V.
+        This property can be set.""",
+        validator=truncated_range,
+        values=[-10.5, 10.5]
+    )
+    # For consistency with other lock-in instrument classes
+    dac3 = aux_out_3
+
+    aux_out_4 = Instrument.control(
+        "AUXV? 3", "AUXV 4, %f",
+        """ A floating point property that controls the output of Aux output 4 in
+        Volts, taking values between -10.5 V and +10.5 V.
+        This property can be set.""",
+        validator=truncated_range,
+        values=[-10.5, 10.5]
+    )
+    # For consistency with other lock-in instrument classes
+    dac4 = aux_out_4
+
+    aux_in_1 = Instrument.measurement(
+            "OAUX? 0",
+            """ Reads the Aux input 1 value in Volts with 1/3 mV resolution. """
+    )
+    # For consistency with other lock-in instrument classes
+    adc1 = aux_in_1
+
+    aux_in_2 = Instrument.measurement(
+        "OAUX? 1",
+        """ Reads the Aux input 2 value in Volts with 1/3 mV resolution. """
+    )
+    # For consistency with other lock-in instrument classes
+    adc2 = aux_in_2
+
+    aux_in_3 = Instrument.measurement(
+        "OAUX? 2",
+        """ Reads the Aux input 3 value in Volts with 1/3 mV resolution. """
+    )
+    # For consistency with other lock-in instrument classes
+    adc3 = aux_in_3
+
+    aux_in_4 = Instrument.measurement(
+        "OAUX? 3",
+        """ Reads the Aux input 4 value in Volts with 1/3 mV resolution. """
+    )
+    # For consistency with other lock-in instrument classes
+    adc4 = aux_in_4
+
+    gettimebase = Instrument.measurement(
+        "TBSTAT?",
+        """Returns the current 10 MHz timebase source."""
+    )
+    extfreqency = Instrument.measurement(
+        "FREQEXT?",
+        """Returns the external frequency in Hz."""
+    )
+    detectedfrequency = Instrument.measurement(
+        "FREQDET?",
+        """Returns the actual detected frequency in HZ."""
+    )
+    get_signal_strength_indicator = Instrument.measurement(
+        "ILVL?",
+        """Returns the signal strength indicator."""
+    )
+    get_noise_bandwidth = Instrument.measurement(
+        "ENBW?",
+        """Returns the equivalent noise bandwidth, in hertz."""
+    )
+    # Display Commands
+    front_panel = Instrument.control(
+        "DBLK?", "DBLK %i",
+        """Turns the front panel blanking on(i=0) or off(i=1).""",
+        validator=strict_discrete_set,
+        values=ON_OFF_VALUES,
+        map_values=True
+    )
+    screen_layout = Instrument.control(
+        "DLAY?", "DLAY %i",
+        """A integer property that Sets the screen layout to trend(i=0), full strip chart history(i=1), half strip chart history(i=2),
+        full FFT(i=3), half FFT(i=4) or big numerical(i=5).""",
+        validator=strict_discrete_set,
+        values=SCREEN_LAYOUT_VALUES,
+        map_values=True
+    )
+
+    def screenshot(self):
+        self.write("DCAP")

```

Comment:
I'm wondering, where is this screenshot stored. Maybe worth mentioning this in a docstring.